xen/arch/arm/coproc/coproc.c

```diff
@@ -183,9 +183,19 @@ static int coproc_attach_to_domain(struct domain *d,
         goto out;
     }
 
+    ret = iommu_assign_coproc(d, coproc->dev);
```

Comment:
In general an iommu is not mandatory for a coproc functionality nor for scf function.
This rather depends on platform (SoC) if there is an iommu between a coproc and bus.
But given that we have no platform layer in scf yet, let iommu calls live here for now.